### ISSUE TITLE
👋

### ISSUE THREAD

**Issue by Chiroyce1** [illogicalapple/chiroyce3#1]

**Comment by illogicalapple** [comment]
hey

**Comment by Chiroyce1** [comment]
so are you planning to fix the logo or the entire page?

**Comment by illogicalapple** [comment]
i added you could you upload a `favicon.ico` into `public`

im fixing the entire page >:D

**Comment by Chiroyce1** [comment]
> i added you could you upload a `favicon.ico` into `public`

done

**Comment by illogicalapple** [comment]
> > i added you could you upload a `favicon.ico` into `public`
> 
> done

cool what's the font

**Comment by Chiroyce1** [comment]
wait i added a png instead of ico lemme fix that


btw system default fotns are enough imo but Montserrat is cool

**Comment by illogicalapple** [comment]
(by the way before you ask im doing this cause im bored)

**Comment by Chiroyce1** [comment]
Also ive never used vue before so could you redirect the `favicon.ico` file to `favicon.png`

> (by the way before you ask im doing this cause im bored)

makes sense lol

**Comment by illogicalapple** [comment]
> Also ive never used vue before so could you redirect the `favicon.ico` file to `favicon.png`
> 
> > (by the way before you ask im doing this cause im bored)
> 
> makes sense lol

no i couldn't you can use a png to ico converter
`favicon.ico` just tells the browser what the favicon is no code needed

**Comment by Chiroyce1** [comment]
eee im too lazy to convert the png to ico but ok

**Comment by illogicalapple** [comment]
this website is not mobile friendly: https://illogicalapple.com/blog/css-vh-bugs but ill fix that later

**Comment by Chiroyce1** [comment]
why the converter taking so long aa
<img width="404" alt="image" src="https://user-images.githubusercontent.com/97374054/162764554-c7f4de5d-190a-4dbf-95d5-d5dc6dc33a83.png">

**Comment by Chiroyce1** [comment]
wait i just saw that you pushed a commit that fixes this ok

**Comment by illogicalapple** [comment]
lollllllll

**Comment by illogicalapple** [comment]
[ having a favicon.ico is good practice tho ]

**Comment by Chiroyce1** [comment]
you have a point, but that's just the default, a .png will work on all modern browsers if specified

**Comment by illogicalapple** [comment]
yes but it'll have an ugly 404 error [the browser tries to fetch a favicon.ico]

**Comment by Chiroyce1** [comment]
> yes but it'll have an ugly 404 error [the browser tries to fetch a favicon.ico]

no..
<img width="806" alt="image" src="https://user-images.githubusercontent.com/97374054/162765551-eee333d1-b291-4e76-9ef0-06a85a901938.png">

(in the console of [my site](https://chiroyce1.github.io/))

**Comment by illogicalapple** [comment]
> > yes but it'll have an ugly 404 error [the browser tries to fetch a favicon.ico]
> 
> no.. <img alt="image" width="806" src="https://user-images.githubusercontent.com/97374054/162765551-eee333d1-b291-4e76-9ef0-06a85a901938.png">
> 
> (in the console of [my site](https://chiroyce1.github.io/))

just chrome ig then

**Comment by illogicalapple** [comment]
im bored im gonna play video games
